docs/tutorials/api/makers.ipynb

```diff
@@ -287,21 +288,21 @@
     "    skydir=(83.63, 22.01), axes=[energy_axis], width=5, binsz=0.02\n",
     ")\n",
     "\n",
-    "maker = MapDatasetMaker()\n",
+    "dataset_maker = MapDatasetMaker()\n",
     "safe_mask_maker = SafeMaskMaker(\n",
     "    methods=[\"aeff-default\", \"offset-max\"], offset_max=\"3 deg\"\n",
     ")\n",
     "\n",
-    "stacked = MapDataset.create(geom)\n",
+    "global_dataset = MapDataset.create(geom)\n",
```

Comment:
why the change to `global_dataset`? `stacked` is quite explicit no?

Reply:
changed back to stacked